test/regress/cli/regress1/bags/proj-issue584.smt2

```diff
@@ -0,0 +1,6 @@
+(set-logic ALL)
+(set-info :status sat)
+(set-option :sygus-rr-synth-input true)
```

Comment:
This option does not exist anymore, can you remove it?

Reply:
Done